src/james.crypto.md5.clss.pas

```diff
@@ -52,13 +57,37 @@ implementation
 { TMD5Stream }
 
 function TMD5Stream.GetStream: IDataStream;
+{$IFNDEF FPC}
+  function getMd5HashString(Value: string): string;
```

Comment:
That was/is my intention. I just thought of creating unit tests first to make sure this algorithm can be used (before doing the PR), but since the PR has been made, I can also create the new class first, and add an unit test to it.